lib/tasks/mastodon.rake

```diff
@@ -47,7 +47,7 @@ namespace :mastodon do
     confirm = STDIN.gets.chomp
     puts
 
-    if confirm.casecmp?('y')
+    if confirm.casecmp('y')
```

Comment:
Correct me if I'm wrong but casecmp returns -1, 0 or 1. You should append `zero?` method for proper comparison

Reply:
Certainly. Need `zero?`